documentation/secure_storage.md

```diff
@@ -150,7 +150,7 @@ per-device keys for different subsystems using the same algorithm as we
 generate the SSK; An easy way to generate different per-device keys for
 different subsystems is using different static strings to generate the keys.
 
-### Trusted Application Storage Key (TKS)
+### Trusted application Storage Key (TSK)
```

Comment:
Application: Grammatically you're right, but it seems like we always write TA as **T**rusted **A**pplication.

TSK: Thanks!